doc/user/mirrors.md

```diff
@@ -68,16 +57,16 @@ If `--full` is specified, the official image will be cloned intact.
 ### 3. Platform limitation
 
 If you only want to clone packages for a particular platform, you can use `-os` and `-arch` to qualify:
-- `tiup mirros <target-dir> ---os=linux`
-- Just want to clone amd64 architecture: `tiup mirros <target-dir> --arch=amd64`
-- Just want to clone linux/amd64: `tiup mirros <target-dir> --os=linux --arch=amd64`
+- `tiup mirror clone <target-dir> ---os=linux`
+- Just want to clone amd64 architecture: `tiup mirror <target-dir> --arch=amd64`
+- Just want to clone linux/amd64: `tiup mirror <target-dir> --os=linux --arch=amd64`
 
 ### 4. Component version limited
 
 If you want to clone only one version of a component and not all versions, use `--<component>=<version>` to qualify, for example "
-- Just want to clone the v4 version of tidb: `tiup mirrors <target-dir> --tidb v4`
-- Just want to clone the v4 version of tidb, and all versions of tikv: `tiup mirros <target-dir> --tidb v4 --tikv all` 
-- Clone specific versions of all components that start a cluster: `tiup mirrors <target-dir> v4.0.0-rc`
+- Just want to clone the v4 version of tidb: `tiup mirror <target-dir> --tidb v4`
+- Just want to clone the v4 version of tidb, and all versions of tikv: `tiup mirror <target-dir> --tidb v4 --tikv all` 
+- Clone specific versions of all components that start a cluster: `tiup mirror <target-dir> v4.0.0-rc`
```

Comment:
I think these examples should all be `tiup mirror clone xxx`?

Reply:
updated. PTAL